package.json

```
@@ -1,6 +1,8 @@
 {
   "name": "grafana-metricsdrilldown-app",
   "version": "1.0.0-4",
+  "author": "Grafana",
+  "license": "Apache-2.0",
```

To reflect our actual `LICENSE`:

```suggestion
  "license": "AGPL-3.0",
```